src/command_modules/azure-cli-network/azure/cli/command_modules/network/mgmt_nic/azuredeploy.json

```diff
@@ -98,23 +121,45 @@
         "description": "Tags object."
       }
     },
+    "useDnsSettings": {
+      "type": "string",
+      "defaultValue": "false",
+      "allowedValues": [ "true", "false" ],
```

Comment:
It would, but I have to use it as a key in a variable dictionary, and I can't use boolean values for that. 

---
In reply to: [73071481](https://github.com/Azure/azure-cli/pull/592#discussion_r73071481) [](ancestors = 73071481)